llvm-runtimes/compiler-rt-sanitizers/compiler-rt-sanitizers-22.0.0.9999.ebuild

```diff
@@ -117,6 +119,19 @@ src_configure() {
 		local -x CC=${CHOST}-clang
 		local -x CXX=${CHOST}-clang++
 		strip-unsupported-flags
+
+		# The full clang configuration might not be ready yet. Use the partial
+		# configuration files that are guaranteed to exist even during initial
+		# installations and upgrades.
+		local flags=(
+			--no-default-config
+			"--config=${ESYSROOT}/etc/clang/${LLVM_MAJOR}/gentoo-rtlib.cfg"
+			"--config=${ESYSROOT}/etc/clang/${LLVM_MAJOR}/gentoo-stdlib.cfg"
+			"--config=${ESYSROOT}/etc/clang/${LLVM_MAJOR}/gentoo-linker.cfg"
+		)
+		append-cflags "${flags[@]}"
+		append-cxxflags "${flags[@]}"
+		append-ldflags "${flags[@]}"
```

Comment:
Please don't use these functions, and instead inline `local CFLAGS="${CFLAGS} ..."` and so on (the key point is in keeping changes scoped and not polluting the env).

Reply:
Done